app/models.py

```diff
@@ -6,7 +6,7 @@
 cursor = connection.cursor()
 
 def table():
-	connection= psycopg2.connect(host = 'localhost', user = 'postgres', password = 'milamish8', dbname = 'stack')
+	connection = psycopg2.connect(host = 'localhost', user = 'postgres', password = 'milamish8', dbname = 'stack')
```

Comment:
there should be no space between function parameters and their default values. i.e `...connect(host='localhost', user='postgres', ...)`